sda-commons-starter-web/src/main/resources/org/sdase/commons/spring/boot/web/monitoring/monitoring.properties

```
@@ -20,5 +20,6 @@ management.endpoint.health.show-components=always
 # Metrics
 management.endpoints.web.path-mapping.prometheus=metrics/prometheus
 management.metrics.web.server.request.autotime.enabled=true
-management.endpoint.prometheus.enabled=true
-management.endpoint.metrics.enabled=true
+management.endpoint.prometheus.access=read_only
+management.endpoint.metrics.access=read_only
+management.endpoint.availableEndpoints.access=read_only
```

In the notes you linked it's `read-only`.

```suggestion
management.endpoint.prometheus.access=read-only
management.endpoint.metrics.access=read-only
management.endpoint.availableEndpoints.access=read-only
```